### ISSUE TITLE
License Invalid Message

### ISSUE THREAD

**Issue by rbae** [altimesh/hybridizer-basic-samples#34]
I have:  

- Visual Studio 2017 Professional, 15.4.4
- CUDA 9.1 installed and Visual Studio Integration enabled.  I am able to compile the CUDA samples

I receive a license for Hybridizer, but it tells me that the license is invalid when I try to validate in the Hybridizer License Settings dialog.  It shows a License Type of ERROR.

![image](https://user-images.githubusercontent.com/9489566/35494474-bc2daf18-046f-11e8-944d-732916d63960.png)

Is there anything that I'm missing?

Thanks,

-Ryan

**Comment by JeWaVe** [comment]
Thank you for your interest in Hybridizer. 

Did you request and validate the license on the same machine?

**Comment by rbae** [comment]
Yes - request and validate was on the same machine.   The email with the license was received on a different machine (MacOS)... not sure if that makes a difference.

**Comment by JeWaVe** [comment]
It shouldn't. Could you please send me the text license to my email address so I make further investigations? 
Regis dot portalez at altimesh dot com 
Thanks

**Comment by JeWaVe** [comment]
Looks like we have a time zone issue in our license creation/validation process. As soon as confirmed I'll close this issue and create a new one for timezone

**Comment by JeWaVe** [comment]
Issue should be resolved. 
Could you try again to request and validate a license?

**Comment by JeWaVe** [comment]
You're very welcome. We had a time zone issue in our code. Thanks for confirmation
